tests/settings/ConfigurationGatheringTest.py

```diff
@@ -101,13 +101,50 @@ def test_default_coafile_parsing(self):
 
         Constants.system_coafile = tmp
 
+    def test_capture_warnings_for_default_coafile(self):
+        """
+        This test case is temporary.
+        """
+        tmp = Constants.system_coafile
+
+        Constants.system_coafile = os.path.abspath(os.path.join(
+            os.path.dirname(os.path.realpath(__file__)),
+            'section_manager_test_files',
+            'default_coafile'))
+
+        logger = logging.getLogger(__name__)
+        with self.assertLogs(logger, 'WARNING') as log:
```

Comment:
Here `assertLog(boolean assertion,String msg)`in If assertion parameter is false, then logs message as an error statement. Here ` String` will `Log a message object with the ERROR Level`. and you can check [level](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Level.html#ERROR) 
Here you will find there is not any `level-WARNING`

Here you can [check](https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Category.html)